### DISCUSSION TITLE
Is this a 1.20 feature or just a minor release feature of 1.19?

### DISCUSSION THREAD

**Discussion by hexagonstudios0** [Mojang/minecraft-editor#115] [Q&A] [answered]
Is this a 1.20 feature or just a minor release feature of 1.19?

I'd like to know because I am going to be creating worlds with this tool.

**Comment by conmaster2112** [comment]
Minecraft Editor Mode is in Alpha versions so it will not be released in 1.20 stable, hopefully since v4.0 comes out

**Comment by minecraftchloe** [comment, marked as answer]
Great question! Thanks for asking! 

Bedrock Editor is going to exist exclusively in **Preview** builds for a long time, and won't be exposed in stable/retail until we reach v1.0. We are currently in v0.3.5, with our next large feature release target being v0.4: The Editor API Update, which will expose the framework for building custom Editor building tools, which we are calling "Editor Extensions." 

If you're going to be creating worlds with the Editor, exercise caution with utilizing functionalities in your build that are exclusive to Preview or experimental and may not work if not yet released to stable/retail.
